docs/component_branding.md

```diff
@@ -22,12 +26,43 @@ To add colours to a component, modify the component to follow the example below.
   </div>
 
   <a href="#" class="<%= brand_helper.color_class %>">
-    Example element that requires coloured text
+    Obscure use case where a link requires coloured text
   </a>
 </div>
 ```
 
-Note that the helper must be called for each element that needs a border or link colour applying. This allows for flexibility if one is required but not the other.
+The above will style borders with colours from the design system automatically, as we have a SASS mixin for this in `_brand_colours.scss` that will map the class to its colour in the design system.
+
+If your use case fits one of these scenarios:
```

Comment:
I think for the list below this line to format correctly there needs to be an empty line between the two?

Reply:
If it looks fine then that's fine 👍 Wasn't able to check myself.